modules/thrift/misc/hello.thrift

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2022 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+service Hello {
+    void ping();
+    string echo(1: string msg);
+    i32 counter();
```

Comment:
Seems like this should be in a sample in the `samples/` directory

Reply:
It's used by both samples and tests